test/test_masamune.rb

```diff
@@ -130,11 +122,11 @@ def test_find_symbols
     CODE
 
     msmn = Masamune::AbstractSyntaxTree.new(string_symbols)
-    assert msmn.symbols.size == 2
-    assert msmn.string_symbols.size == 1
-    assert msmn.string_symbols.first[:line_number] == 1
-    assert msmn.string_symbols.first[:index_on_line] == 2
-    assert msmn.string_symbols.first[:token] == "foo_bar"
+    assert_equal msmn.symbols.size, 2
+    assert_equal msmn.string_symbols.size, 1
+    assert_equal msmn.string_symbols.first.line_number, 1
+    assert_equal msmn.string_symbols.first.token_location.start_column, 2
+    assert_equal msmn.string_symbols.first.token, "foo_bar"
```

Comment:
By the way, technically the call structure is `assert_equal expected, actual` otherwise the generated diffs on failure look off. So most of the `assert_equal`s would have their arguments reordered like this:

```suggestion
    assert_equal 2, msmn.symbols.size
    assert_equal 1, msmn.string_symbols.size
    assert_equal 1, msmn.string_symbols.first.line_number
    assert_equal 2, msmn.string_symbols.first.token_location.start_column
    assert_equal "foo_bar", msmn.string_symbols.first.token
```

Reply:
Updated in https://github.com/gazayas/masamune-ast/pull/60/commits/a5d8c1ca3bd0512e72be6b220f38f980a11230cd